internal/cmd/lint.go

```diff
@@ -63,3 +103,19 @@ func NewLintCmd() runner.SubCommand {
 	}
 	return &LintCmd{cmd: cmd}
 }
+
+// isDirectory determines if a file represented
+// by `path` is a directory or not
+func isDirectory(path string) (bool, error) {
+	fileInfo, err := os.Stat(path)
+	if err != nil {
+		return false, err
+	}
+
+	return fileInfo.IsDir(), err
+}
+
+func isYaml(path string) bool {
+	ext := filepath.Ext(path)
+	return ext == ".yaml" || ext == ".yml"
+}
```

Comment:
I wonder if we should try to Marshal the files and bail out in case of errors, as in inspect each file trying to identify a Tekton resource based on its contents instead of the extension only.

In general, checking the extension is a good way to start 👍

Reply:
@otaviof yeah, based on https://github.com/openshift-pipelines/tektoncd-catalog/pull/34 (and the error we can get), this is probably the way forward. That way we know it's a k8s/tekton resource, and we skip otherwise.